runtime/jcl/unix/syshelp.c

```diff
@@ -215,7 +215,7 @@ jobject getPlatformPropertyList(JNIEnv * env, const char *strings[], int propInd
 	strings[propIndex++] = "java.io.tmpdir";
 	strings[propIndex++] = getTmpDir(env, &envSpace);
 
-	if (JAVA_SPEC_VERSION < J2SE_V12) {
+	if (JAVA_SPEC_VERSION <= J2SE_V11) {
```

Comment:
This was wrong before (it should have said `JAVA_SPEC_VERSION < 12`); that would continue to be fine.

Reply:
Fixed.